src/Symfony/Component/JsonPath/JsonCrawler.php

```diff
@@ -150,8 +172,8 @@ private function evaluateBracket(string $expr, mixed $value): array
             }
 
             $result = [];
-            foreach (explode(',', $expr) as $index) {
-                $index = (int) trim($index);
+            foreach (preg_split('/\s*,\s*/', $expr) as $indexStr) {
```

Comment:
this could be possessive quantifiers as well.

Reply:
Alternatively, keep the `explode` as you are trimming strings below anyway. Maybe it is more efficient.